### PR TITLE
Finished adding all new boards

### PR DESCRIPTION
**Pull request by makermelissa**
This adds the rest of the missing boards under downloads and a couple of new Blinka boards. I also changed the HDMI to HDMI/DisplayPort for the feature name on a bunch of Blinka boards.

### CODE REVIEW

**Comment by makermelissa** [comment]
Could I get a review on this soon? Some duplicate PRs are starting to appear. It was tested locally as I made these changes and they pass the checking script, so even though there's 100+ files, it's fairly straightforward stuff.

File: _board/hardkernel_odroid_go.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+family: esp32
+downloads_display: true
+features:
+  - Battery Charging
+  - Bluetooth/BTLE
+  - Wi-Fi
+  - Display
+---
+
+**The original Odroid Go has been continued**
```
Comment (dhalbert) [inline _board/hardkernel_odroid_go.md:19]: ?
```suggestion
**The original Odroid Go has been discontinued**
```

**Review by dhalbert — CHANGES_REQUESTED** [review]
Thanks! Just one potential typo. I did not read all the descriptions incredibly thoroughly.

**Review by dhalbert — APPROVED** [review]
:+1:

**Pull request merged**